apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -208,43 +210,77 @@ export default class ChangeAction extends BaseRushAction {
   /**
    * Asks all questions which are needed to generate changelist for a project.
    */
-  private _askQuestions(packageName: string): Promise<IChangeInfo> {
+  private _askQuestions(packageName: string): Promise<IChangeInfo | undefined> {
     console.log(`${os.EOL}${packageName}`);
-    const bumpOptions: { [type: string]: string } = this._getBumpOptions(packageName);
+    const comments: string[] | undefined = this._changeComments.get(packageName);
+    if (comments && comments.length) {
+      console.log(`Found existing comments:`);
+      comments.forEach(comment => {
+        console.log(`    > ${comment}`);
+      });
+      return this._prompt({
+        name: 'appendComment',
+        type: 'list',
+        default: 'skip',
+        message: 'Append to existing comments or skip:',
+        choices: [
+          {
+            'name': 'Skip',
+            'value': 'skip'
+          },
+          {
+            'name': 'Append',
+            'value': 'append'
+          }
```

Comment:
It may be easier for us to just display the existing comment as the default value for the prompt on line 251. That way it is super straightforward to edit

Reply:
Right now there is no good edit experience

---
In reply to: [147835928](https://github.com/Microsoft/web-build-tools/pull/418#discussion_r147835928) [](ancestors = 147835928)

Reply:
Improving edit experience is out of scope for this change.

---
In reply to: [147840641](https://github.com/Microsoft/web-build-tools/pull/418#discussion_r147840641) [](ancestors = 147840641,147835928)